src/full/Agda/Interaction/BasicOps.hs

```diff
@@ -1132,8 +1132,19 @@ introTactic pmLambda ii = do
       r <- splitLast CoInductive tel pat
       case r of
         Left err -> return []
-        Right cov ->
-           mapM showUnambiguousConName $ concatMap (conName . scPats) $ splitClauses cov
+        Right cov -> 
+           mapM conWithArgs $ map scPats $ splitClauses cov
+      where
+        conWithArgs x = do
+           let [ch] = conHead x
```

Comment:
Partiality needs to throw `__IMPOSSIBLE__`. 
With above change, you could have
```
let ch = fromMaybe __IMPOSSIBLE__ $ conName x
```

Reply:
Looking back at what was here before, I think that filtering out `Nothing`s, will more closely match previous behavior. Unless we can safely assume that there is always `I.ConP` in the pattern (but then we could skip `Maybe` altogether in the `conName`)